src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -233,7 +247,47 @@ const ProjectActions = ({
       <ConfirmProceedDialog
         body={
           <p className="typography-body1">
-            {deleteMessage}
+            {
+              savedSearch?.is_part_of_feeds ?
+                <>
+                  <FormattedHTMLMessage
+                    defaultMessage="Are you sure? This is shared among all users of <strong>{teamName}</strong>. After deleting it, no user will be able to access it.<br /><br />"
+                    description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
+                    id="projectActions.deleteMessageWarning"
+                    tagName="p"
+                    values={{
+                      teamName: savedSearch?.team ? savedSearch.team.name : '',
+                    }}
+                  />
+                  <Alert
+                    content={
+                      <ul className="bulleted-list">
+                        {savedSearch?.feeds?.edges.map(feed => (
+                          <li key={feed.node.id}>{feed.node.name}</li>
+                        ))}
+                      </ul>
+                    }
+                    title={
+                      <FormattedHTMLMessage
+                        defaultMessage="<strong>Deleting list will result in no content for the following shared feeds:</strong>"
+                        description="Warning displayed on edit feed page when no list is selected."
+                        id="saveFeed.deleteCustomListWarning"
+                      />
+                    }
+                    variant="warning"
+                  />
+                </>
+                :
+                <FormattedHTMLMessage
+                  defaultMessage="Are you sure? This is shared among all users of <strong>{teamName}</strong>. After deleting it, no user will be able to access it."
+                  description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
+                  id="projectActions.deleteMessage"
```

Comment:
this was originally `savedSearch.deleteMessage`, no?

Reply:
Yes and updated to `savedSearch.deleteMessage`